src/Microsoft.DotNet.Configurer/NuGetCacheSentinel.cs

```diff
@@ -23,7 +23,7 @@ private string NuGetCachePath
             {
                 if (string.IsNullOrEmpty(_nugetCachePath))
                 {
-                    _nugetCachePath = PackageDependencyProvider.ResolvePackagesPath(null, null);
+                    _nugetCachePath = NuGetPathContext.Create(new NullSettings()).UserPackageFolder;
```

Comment:
Will this take into account `NUGET_PACKAGES` and other package location controls?